tests/page/page-keyboard-layouts.spec.ts

```diff
@@ -0,0 +1,363 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
```

Comment:
this should be just Microsoft copyright. E.g. from [here](https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/client/fetch.ts#L2).